content/docs/getting-started/install-github-app/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Installing the GitHub App
+publishedDate: '2022-03-21T21:00:00.0Z'
+description: How to give Roadie the access it needs to your GitHub org.
+---
+
+## Introduction
+
+This tutorial will guide you through the steps required to connect Roadie to your GitHub org.
```

Comment:
Should we have a page that gives you a bit of orientation.

I feel like this page should be called something like: "Configuring GitHub as a catalog source" or something like that, then when Jussi is done, it'd have a similar page for Bitbucket.

Also it feels like this intro could say something like:

"This tutorial will guide you through the steps required to configure Backstage to index your GitHub Organization projects and teams by installing a GitHub App."

Reply:
That's pretty much the direction I was going here. I wanted to break out the individual pages so I can link to them from a docs homepage. That will come in a different PR.